doc/develop/acq.rst

```diff
@@ -372,3 +372,70 @@ SPARC alignment:
 In mirror alignment mode, during background acquisition, the SEM is stopped to
 automatically activate the blanker (which will happen only if it's in the automatic setting).
 
+Optical Path Manager
+====================
+The goal of the Optical Path Manager is to the position actuators affecting the
+light path such that they are in the right configuration for a given acquisition type.
+It is found in ``acq.path.OpticalPathManager`` and is typically used as a singleton.
+The GUI takes care of instantiating it at init, and then it can be accessed via
+``MainGUIData.opm``. In practice, this is almost exclusively used on the SPARC.
+
+
+It has one important method:
+
+.. py:method:: setPath(mode, detector=None)
+
+   Sets all the necessary components of the optical path to the positions
+   required to achieve the requested mode.
+
+   :param path: The stream or the optical path mode
+   :type path: stream.Stream or str
+   :param detector:
+      When the `path` is a str, this allows
+      to define which detector will be targeted on this path. That
+      is useful in case the mode can be used with multiple detectors (eg,
+      ``fiber-align`` on a SPARC with multiple spectrometers). When path is a
+      Stream, the ``Stream.detector`` is always used.
+   :type detector: Component or None
+   :return: a Future allowing to follow the status of the path update.
+   :rtype: Future
+   :raise: ValueError if the given mode does not exist
+           IOError if a detector is missing
+
+The list of supported `paths` (ie, optical modes) can be found in the ``acq.path`` module.
+At runtime, they are limited by the microscope type and the available components.
+
+In terms of implementation, for each optical mode, there are two types of actuators:
+
+ * The path selectors which, typically, are mirrors with 2 positions. Their positions
```

Comment:
I think this "affects" property and the way the optical path is calculated could need some more information. If you have heard about it before, it is fine, but if you never have heard about it, it will be difficult to understand how the optical path is derived. You explained this nicely yesterday in your talk. 
Maybe it is not the right place here, but somewhere in this section you could add information, that the affects needs to be defined in the microscope file, and that you start from the detector towards the actuators (if I understood this correctly).